blackbox/docs/sql/statements/create-repository.rst

```diff
@@ -357,6 +357,12 @@ A repository type that stores its snapshots on the Azure Storage service.
   The location mode for storing data on the Azure Storage.
   Note that if you set it to ``secondary_only``, it will force readonly to true.
 
+Azure client settings
+.....................
+
+All the setting values are specified via ``azure.client.`` in the crate.yaml
```

Comment:
Is there a reason why this section was introduced? It is also not correct.

Reply:
@kovrus I will have to look at this together with @mechanomi.